calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_models/ceres_camera_model.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List
+from typing import Optional
+
+from intrinsic_camera_calibrator.camera_models.camera_model import CameraModel
+from intrinsic_camera_calibrator.camera_models.opencv_camera_model import OpenCVCameraModel
+import numpy as np
+
+os.environ["GLOG_minloglevel"] = "2"  # cSpell:ignore minloglevel
+from ceres_intrinsic_camera_calibrator.ceres_intrinsic_camera_calibrator_py import (  # noqa: E402
+    calibrate,
+)
+
+
+class CeresCameraModel(CameraModel):
+    """Basic Ceres camera model class."""
+
+    def __init__(
+        self,
+        k: Optional[np.array] = None,
+        d: Optional[np.array] = None,
+        height: Optional[int] = None,
+        width: Optional[int] = None,
+    ):
+        super().__init__(k, d, height, width)
+        self.radial_distortion_coefficients = 3
+        self.rational_distortion_coefficients = 3
+        self.use_tangential_distortion = True
+        self.pre_calibration_num_samples = 6
+        self.regularization_weight = 0.001
```

Comment:
having these values defined here makes it difficult to understand. Everything should be loaded from the `Parameter` class

Reply:
nit:

Regarding typing:
`int_value: int = None`
should be
`in_value: Optional[int] = None`

Reply:
Addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/b6168317810cfd8abf813aa2d7598b9940111530